cmake/testing.cmake

```diff
@@ -247,53 +249,52 @@ if(F3D_MODULE_EXTERNAL_RENDERING)
   f3d_test(NAME TestVersionExternal ARGS --version REGEXP "External rendering module: ON")
 endif()
 
-if(F3D_MODULE_EXODUS)
-  f3d_test(NAME TestExodus DATA disk_out_ref.ex2 ARGS -s --camera-position=-11,-2,-49 DEFAULT_LIGHTS)
-  f3d_test(NAME TestGenericImporterAnimation DATA small.ex2 DEFAULT_LIGHTS)
+if(F3D_PLUGIN_BUILD_EXODUS)
+  f3d_test(NAME TestExodus DATA disk_out_ref.ex2 ARGS --load-plugins=exodus -s --camera-position=-11,-2,-49 DEFAULT_LIGHTS)
```

Comment:
what happens if you load a static plugin that has already been autoloaded ? I dont think a warning is needed, should bi silent.

Reply:
I don't think it should be different with dynamic plugin though. Loading a library twice is well supported by all OS. They will just return the same handle.  
So what about changing the warning in a debug message in all cases?

Reply:
good, maybe a log ?

Reply:
you mean `log::info`? Do we want to display the message by default? If not, then it should be a `log::debug`

Reply:
right, my bad.